soc/riscv/qemu_virt_riscv/qemu_virt_riscv32/Kconfig

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 Antmicro <www.antmicro.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_QEMU_VIRT_RISCV32
+	select CPU_HAS_FPU
+	select RISCV_ISA_RV32I
+	select RISCV_ISA_EXT_ZICSR
+	select RISCV_ISA_EXT_ZIFENCEI
```

Comment:
qemu_virt_riscv32 and qemu_virt_riscv64 should probably become one board and use variant to specify which one is being built where no variant = 32-bit and variant "64" = 64-bit, thoughts @gmarull @aescolar ?

Reply:
I don't have a strong preference on this. But if we go ahead and convert them to a single board with variants, I suppose that `qemu_virt_riscv32e` should also be included in the process?

Reply:
initial feeling is that it's the SoC which is either 32 bit or 64bit, and thus you select the SoC of the arch you want.
Thus the board indirectly select 32 / 64 bit based on the SoC it selects.

Not the board which should directly select 32 or 64 bit.